gravitee-am-model/src/main/java/io/gravitee/am/model/Domain.java

```diff
@@ -396,6 +399,14 @@ public void setSaml(SAMLSettings saml) {
         this.saml = saml;
     }
 
+    public List<Language> getLanguages() {
+        return languages;
```

Comment:
Can this default to an empty list? Either `Collections.emptyList()` or List.of()`?